cmd/influx/cli/cli.go

```diff
@@ -777,38 +778,103 @@ func (c *CommandLine) writeJSON(response *client.Response, w io.Writer) {
 	fmt.Fprintln(w, string(data))
 }
 
+func tagsEqual(prev, current map[string]string) bool {
+	if prev == nil && current == nil {
+		return true
+	}
+	if len(prev) != len(current) {
```

Comment:
You can probably just remove this too. I checked the source of `DeepEqual` and one of the first things it does it check the [length of two maps](https://golang.org/src/reflect/deepequal.go?s=5823:5860#L110).